av/container/core.pyx

```diff
@@ -251,7 +333,7 @@ cdef class Container(object):
 def open(file, mode=None, format=None, options=None,
          container_options=None, stream_options=None,
          metadata_encoding='utf-8', metadata_errors='strict',
-         buffer_size=32768, timeout=None):
+         buffer_size=32768, timeout=None, io_open=None):
```

Comment:
This `io_open` argument needs a docstring explaining what the expected signature is, and how to use it.

Reply:
Added in https://github.com/PyAV-Org/PyAV/pull/718/commits/d84cc95b17d8dae6ac2b3329851d00fc7cda0d76